skills/changeset/references/patterns.md

```
@@ -0,0 +1,175 @@
+- 기존 prop/API 제거 또는 이름 변경
+- 기존 동작의 breaking change
+- 컴포넌트 이름 변경
+- 필수 마이그레이션이 필요한 변경
+
+## 메시지 구조
+
+### patch — 1줄
+
+간결한 변경사항 한 줄:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)